nvflare/apis/fl_exception.py

```diff
@@ -33,3 +33,9 @@ class UnsafeJobError(Exception):
     """Raised when a job is detected to be unsafe"""
 
     pass
+
+
+class UnsafeComponentError(Exception):
+    """Raised when configuration parsing error happens."""
+
+    pass
```

Comment:
What about """Raised when a component in the configuration is detected to be unsafe"""?

Reply:
Changed as suggested.